holochain/holochain-rust
# 81 mdbook

fixes #81 

Provides stubs and docker for mdbook `holochain_101`

changes:

- updated readme
- added docker with mdbook installed
- docker run script serves and live reloads mdbook

to test:
- `. docker/build-mdbook && . docker/run-mdbook`
- once the mdbook is serving, visit `http://localhost:3000` in the browser
- make an edit to a file in `doc/holochain_101` (don't forget to update SUMMARY.md if adding a chapter)
- see that edits appear in the browser with live reload
